### PR TITLE
minor changes based off dev input

### PR DESCRIPTION
**Pull request by jc-berger**
see Jira: https://issues.redhat.com/browse/ACM-9733

### CODE REVIEW

File: business_continuity/backup_restore/backup_arch.adoc [hunk 1]
```diff
@@ -16,11 +16,11 @@ View the following ordered list of the cluster backup and restore process for id
 
 - Exclude all resources in the `MultiClusterHub` namespace. This is to avoid backing up installation resources that are linked to the current hub cluster identity and should not be backed up.
 
-- Back up all custom resource definitions with an API version suffixed by `.open-cluster-management.io`. This suffix indicates that all {product-title-short} resources are backed up.
+- Back up all custom resource definitions with an API version suffixed by `.open-cluster-management.io` and `.hive.openshift.io`. These suffixes indicate that all {product-title-short} resources are backed up.
```
Comment (birsanv) [inline business_continuity/backup_restore/backup_arch.adoc]: replace Back up all custom resource definitions
with Back up all resources

File: business_continuity/backup_restore/backup_arch.adoc [hunk 2]
```diff
@@ -16,11 +16,11 @@ View the following ordered list of the cluster backup and restore process for id
...
 
-- Back up all custom resource definitions from the following API groups: `argoproj.io`, `app.k8s.io`, `core.observatorium.io`, `hive.openshift.io`.
+- Back up all custom resource definitions from the following API groups: `argoproj.io`, `app.k8s.io`, `core.observatorium.io`, `hive.openshift.io`. These API groups are under the resources backup, and all the CRDs from the group, `agent-install.openshift.io`, are under the managed clusters backup.
```
Comment (birsanv) [inline business_continuity/backup_restore/backup_arch.adoc]: replace Back up all custom resource definitions from the following API groups: `argoproj.io`, `app.k8s.io`, `core.observatorium.io`, `hive.openshift.io` and `agent-install.openshift.io`.
with Back up all resources from  `argoproj.io`, `app.k8s.io`, `core.observatorium.io`, `hive.openshift.io`.

Comment (birsanv) [reply business_continuity/backup_restore/backup_arch.adoc]: replace : These API groups are under the resources backup, and all the CRDs from the group, `agent-install.openshift.io`, are under the managed clusters backup.

with: 
These resources are backed up under the `acm-resources-schedule` backup, with the exception of resources from the `agent-install.openshift.io` API group, which are backed up under the `acm-managed-clusters-schedule` backup.

File: business_continuity/backup_restore/backup_arch.adoc [hunk 3]
```diff
@@ -16,11 +16,11 @@ View the following ordered list of the cluster backup and restore process for id
...
+- Back up all custom resource definitions from the following API groups: `argoproj.io`, `app.k8s.io`, `core.observatorium.io`, `hive.openshift.io`. These API groups are under the resources backup, and all the CRDs from the group, `agent-install.openshift.io`, are under the managed clusters backup. 
 
-- Exclude all custom resource definitions from the following API groups: `admission.cluster.open-cluster-management.io`, `admission.work.open-cluster-management.io`,  `internal.open-cluster-management.io`, `operator.open-cluster-management.io`, `work.open-cluster-management.io`, `search.open-cluster-management.io`, `admission.hive.openshift.io`, `velero.io`.
+- Exclude all custom resource definitions from the following API groups: `internal.open-cluster-management.io`, `operator.open-cluster-management.io`, `work.open-cluster-management.io`, `search.open-cluster-management.io`, `admission.hive.openshift.io`, `proxy.open-cluster-management.io`, `action.open-cluster-management.io`, `view.open-cluster-management.io`, `clusterview.open-cluster-management.io`, `velero.io`.
```
Comment (birsanv) [inline business_continuity/backup_restore/backup_arch.adoc]: replace Exclude all custom resource definitions
with Exclude all resources

File: business_continuity/backup_restore/backup_arch.adoc [hunk 4]
```diff
@@ -16,11 +16,11 @@ View the following ordered list of the cluster backup and restore process for id
...
 
 - Exclude the following custom resource definitions that are a part of the included API groups, but are either not needed or are being recreated by owner resources, which are also backed up: `clustermanagementaddon`, `observabilityaddon`, `applicationmanager`, `certpolicycontroller`, `iampolicycontroller`, `policycontroller`, `searchcollector`, `workmanager`, `backupschedule`, `restore`, `clusterclaim.cluster.open-cluster-management.io`.
```
Comment (birsanv) [inline business_continuity/backup_restore/backup_arch.adoc]: replace Exclude all custom resource definitions
with Exclude all resources

**Review by birsanv — CHANGES_REQUESTED** [review]

File: business_continuity/backup_restore/backup_arch.adoc (hunk 2 above)
Comment (swopebe) [reply business_continuity/backup_restore/backup_arch.adoc]: Let's find a better word for "under" here, as we try to keep direction language out tech doc.

**Review by swopebe — APPROVED** [review]
approve, pending developer changes and one comment for change. @jc-berger

**Review by birsanv — APPROVED** [review]
lgtm thanks!

**Pull request merged**